exporters/aerospike/e2e/e2e_spec.yml

```diff
@@ -0,0 +1,46 @@
+description: |
+  End-to-end tests for Aerospike integration
+
+scenarios:
+  - description: |
+      This scenario will verify that metrics from aerospike prometheus exporter are correcly collected.
+    before:
+      - docker-compose up -d
+    after:
+      - docker-compose down -v
+    integrations:
+      - name: nri-aerospike
+        binary_path: ../target/bin/linux_amd64/nri-aerospike
+        exporter_binary_path: ../target/bin/linux_amd64/aerospike-exporter
+        config:
+          aerospike_cluster_name: aerospike_cluster_name
+          aerospike_db_host: localhost
+          aerospike_db_port: 3000
+          exporter_port: 9145
+          label_type: development
+          label_source: aerospike
+          exporter_files_path: /tmp
+
+    tests:
+      nrqls:
+        - query: "FROM Metric SELECT latest(aerospike_node_stats_client_connections)"
+      entities:
+        - type: "AEROSPIKE_NODE"
```

Comment:
According to the entity-definition (https://github.com/newrelic/entity-definitions/blob/main/definitions/ext-aerospike/definition.yml#L2) we only have an Entity `AEROSPIKE`, so you need to adjust the test to the correct entity name and remove `AEROSPIKE_NODE` and `AEROSPIKE_NAMESPACE`